python/lsst/obs/base/_instrument.py

```diff
@@ -499,21 +525,11 @@ class attribute for curated calibrations corresponding to the
             for timespan, refs in refsByTimespan.items():
                 butler.registry.certify(collection, refs, timespan)
 
-    @abstractmethod
-    def makeDataIdTranslatorFactory(self) -> TranslatorFactory:
+    def makeDataIdTranslatorFactory(self) -> None:
```

Comment:
Is this supposed to be deprecated, or does that happen to the derived classes?

Reply:
Since I removed the abstractmethod I should probably delete the entire method since there is no code to use it any more. I was a bit worried that I should keep a bunch of the gen2to3 classes around with deprecations for any instrument subclasses that implement this method...

Reply:
I think that gen2to3 stuff falls under the "remove in 2022" so should you remove this and remove any implementations that jenkins turns up (if any)?